README.md

```diff
@@ -278,18 +278,19 @@ UDS CLI includes a vendored version of Zarf inside of its binary. To use Zarf, s
 ## Dev Mode
 
 > [!NOTE]  
-> Dev mode is a BETA feature and currently only works with local bundles
+> Dev mode is a BETA feature
 
 Dev mode facilitates faster dev cycles when developing and testing bundles
 
 ```
-uds dev deploy <path-to-bundle-yaml-dir>
+uds dev deploy <path-to-bundle-yaml-dir> | <oci-ref>
 ```
 
 The `dev deploy` command performs the following operations
-- Creates Zarf packages for all local packages in a bundle
+
+- (If local bundle) Creates Zarf packages for all local packages in a bundle
```

Comment:
Let's remove the parentheses from "(If local bundle)" and just use a colon, can also remove the "(If local bundle)" below and tab the "Creates a bundle from the..." over